### PR TITLE
Added pagetitle to renamer.

### LINKED ISSUE #150: Title mask is not using tab/page title and no folder is created [Firefox]
**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox 69.0.2 (64-bit) & 71.0a1 (2019-10-04) (64-bit)
 - Version 4.1.2

**Describe the bug**
Whenever I use *title* either in subfolders or mask field, it does absolutely nothing and no folder is created.

e.g. 
```
Subfolders: *title*
Mask: *idx*.*ext*
```
**Result =** all files appear within Downloads folder and no folder is created

```
Subfolders: 
Mask: *title*/*idx*.*ext*
```
**Result =** all files appear within Downloads folder and no folder is created

**Expected behavior**
A download folder is created using either tab or webpage title.

### PR DESCRIPTION
**Pull request by brownm09**
Adds `*pagetitle*` to the mask options.
Resolves #150.

### CODE REVIEW

File: scripts/gather.ts [hunk 1]
```diff
@@ -103,6 +103,8 @@ class Gatherer {
   }
 
   collectLink(a: HTMLAnchorElement) {
+    const pageTitle = document.title;
+    console.log(`Page title: ${pageTitle}`);
```
Comment (d7415) [inline scripts/gather.ts:107]: This debug code should probably be removed.

Comment (brownm09) [reply scripts/gather.ts:107]: Yup, fixed! Forgot I'd left it in there.

**Comment by d7415** [comment]
> Hi, I'm not super familiar with this contribution model - is there something blocking this from being merged?

It has been merged. GitHub sometimes gets confused if it's not merged from their website.

**Comment by nmaier** [comment]
Yes, thanks for your contribution. It has been merged, with some minor fixes (you accidentally changed the line endings of one of the files)

**Comment by brownm09** [comment]
> > Hi, I'm not super familiar with this contribution model - is there something blocking this from being merged?
> 
> It has been merged. GitHub sometimes gets confused if it's not merged from their website.



> Yes, thanks for your contribution. It has been merged, with some minor fixes (you accidentally changed the line endings of one of the files)

Thank you! I appreciate the minor fixes as well as the responses to my inquiry. I hope to contribute more, as this is one my favorite extensions.

**Pull request closed without merging**